free-programming-books-id.md

```diff
@@ -103,12 +104,13 @@
 
 * [Belajar Python](http://www.belajarpython.com)
 * [Dasar Pemrograman Python](https://www.pythonindo.com/tutorial-python-dasar/)
+* [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan, Google Play Books (account required)
```

Comment:
```suggestion
* [Dasar Pengenalan Pemrograman Python](https://play.google.com/store/books/details/Rolly_Maulana_Awangga_Dasar_dasar_Python?id=YpzDDwAAQBAJ) - Rolly Maulana AwanggaRayhan, Google Play Books *(Membutuhkan akun Google atau email yang valid)*
```

Notes as same as in ... https://github.com/EbookFoundation/free-programming-books/blob/89dca4e2e47f5428011bdfc69495db2476fa99b5/free-programming-books-id.md#L75

Reply:
Resolved in commit: https://github.com/EbookFoundation/free-programming-books/pull/6163/commits/8f4b9003d5310a77d8f901d11a751c5280e8d907